package.json

```diff
@@ -149,6 +149,7 @@
     "uglifyjs-webpack-plugin": "^1.0.0",
     "unused-files-webpack-plugin": "^3.0.0",
     "webpack": "^5.37.0",
+    "webpack-cli": "^4.7.0",
```

Comment:
# Install `webpack-cli`

`yarn add -D webpack-cli`
```
warning " > @cypress/webpack-preprocessor@4.1.5" has incorrect peer dependency "webpack@^4.18.1".
warning " > @testing-library/cypress@5.3.1" has incorrect peer dependency "cypress@^2.1.0 || ^3.0.0 || ^4.0.0".
warning " > babel-loader@7.1.5" has incorrect peer dependency "webpack@2 || 3 || 4".
warning " > extract-text-webpack-plugin@3.0.2" has incorrect peer dependency "webpack@^3.1.0".
warning " > html-webpack-plugin@2.30.1" has incorrect peer dependency "webpack@1 || ^2 || ^2.1.0-beta || ^2.2.0-rc || ^3".
warning " > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
warning " > uglifyjs-webpack-plugin@1.3.0" has incorrect peer dependency "webpack@^2.0.0 || ^3.0.0 || ^4.0.0".
```

Running `yarn-dev-ee` after this produces the following error:
```
[cljs] .[webpack-cli] Failed to load '/Users/offbynone/code/work/metabase/webpack.config.js' config
[cljs] .[webpack-cli] TypeError: webpack.optimize.CommonsChunkPlugin is not a constructor
[frontend]     at Object.<anonymous> (/Users/offbynone/code/work/metabase/webpack.config.js:129:5)
[frontend]     at Module._compile (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
[frontend]     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
[frontend]     at Module.load (internal/modules/cjs/loader.js:928:32)
[frontend]     at Function.Module._load (internal/modules/cjs/loader.js:769:14)
[frontend]     at Module.require (internal/modules/cjs/loader.js:952:19)
[frontend]     at require (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
[frontend]     at loadConfig (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1327:31)
[frontend]     at WebpackCLI.resolveConfig (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1459:44)
[frontend]     at WebpackCLI.createCompiler (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/lib/webpack-cli.js:1837:33)
[frontend]     at Command.<anonymous> (/Users/offbynone/code/work/metabase/node_modules/@webpack-cli/serve/lib/index.js:81:40)
[frontend]     at Command.listener [as _actionHandler] (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:922:31)
[frontend]     at Command._parseCommand (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1503:14)
[frontend]     at Command._dispatchSubcommand (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1443:18)
[frontend]     at Command._parseCommand (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1460:12)
[frontend]     at Command.parse (/Users/offbynone/code/work/metabase/node_modules/webpack-cli/node_modules/commander/index.js:1292:10)
error Command failed with exit code 2.
```

Related reading material:
- [RIP CommonsChunkPlugin](https://gist.github.com/sokra/1522d586b8e5c0f5072d7565c2bee693)
- https://webpack.js.org/plugins/commons-chunk-plugin/
> Warning
The CommonsChunkPlugin has been removed in webpack v4 legato. To learn how chunks are treated in the latest version, check out the [SplitChunksPlugin](https://webpack.js.org/plugins/split-chunks-plugin/).